spec/helpers/membership_applications_helper_spec.rb

```diff
@@ -8,15 +8,6 @@
   end
 
 
-  describe '#member_full_name' do
-    it 'appends first and last with a space inbetween' do
-      assign(:membership_application, create(:membership_application, user: create(:user, first_name: 'Kitty', last_name: 'Kat')))
-      expect(helper.member_full_name).to eq('Kitty Kat')
-    end
-
-  end
-
```

Comment:
It looks like `User` is missing some tests for attribute accessors. I'll start a new PR and reference this one.